### PR TITLE
Add server_time method to kraken exchange

### PR DESCRIPTION
**Pull request by sdeslauriers**
This commit starts implementing the `Kraken` exchange. For now the only complete method is `server_time`. All other methods are dummies to be able to instantiate and test the new class.

### CODE REVIEW

File: exchanges/kraken.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        self.url = 'https://api.kraken.com/0/'
+
+    def server_time(self):
+        """Queries the current server time (unix time stamp)"""
+
+        # Request the server time. This can raise an exception
+        # if there is a connection error or if the request times
+        # out. In addition, raise an exception if the status is not
+        # OK.
+        response = requests.get(self.url + 'public/Time', timeout=self.timeout)
```
Comment (gagdiez) [inline exchanges/kraken.py]: @sdeslauriers. Maybe we should implement a _public_query method? as in: https://github.com/gagdiez/exchanges/blob/cassandra/exchanges/kraken/api.py. All the public requests to kraken follow the same format.

Comment (sdeslauriers) [inline exchanges/kraken.py]: Good idea.

**Comment by gagdiez** [comment]
@sdeslauriers. Maybe we should implement a _public_query method? as in: https://github.com/gagdiez/exchanges/blob/cassandra/exchanges/kraken/api.py. All the public requests to kraken follow the same format.

**Review by gagdiez — COMMENTED** [review]
This commit is ok, but maybe we should abstract the public queries to kraken in a method.

File: exchanges/kraken.py (hunk 1 above)
Comment (sdeslauriers) [reply exchanges/kraken.py]: Good idea!

**Review by gagdiez — APPROVED** [review]
The _public_query method is able to query kraken and handle exceptions. 10/10 would approve.

**Comment by sdeslauriers** [comment]
Commit 167ae337fcdc100aee57ef434f9e15277be94987 adds a private method to access public data.

**Pull request merged**